sqlglot/expressions.py

```diff
@@ -3673,25 +3673,57 @@ def table_(table, db=None, catalog=None, quoted=None, alias=None) -> Table:
     )
 
 
-def values(values, alias=None) -> Values:
+def values(
+    values: t.Sequence[t.Tuple[str | Expression, ...]],
+    alias: t.Optional[str] = None,
+    columns_and_types: t.Optional[t.List[str | t.Tuple[str, str | DataType]]] = None,
```

Comment:
make this a dict and call it schema?

Reply:
maybe make it an iterable then

Reply:
did you mean collection (like what we use in `ensure_collection`)?

Reply:
no i meant the type